models/cart/src/custom-line-item/custom-line-item-draft/field-config.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  MoneyDraft,
+  LocalizedStringDraft,
+} from '@commercetools-test-data/commons';
+import {
+  fake,
+  type TModelFieldsConfig,
+  oneOf,
+} from '@commercetools-test-data/core';
+import { customLineItemPriceMode } from '../../cart/constants';
+import {
+  TCustomLineItemDraftRest,
+  TCustomLineItemDraftGraphql,
+} from '../types';
+
+const commonFieldsConfig = {
+  key: fake((f) => f.string.uuid()),
+  name: fake(() => LocalizedStringDraft.random()),
+  quantity: fake((f) => f.number.int(9)),
+  money: fake(() => MoneyDraft.random()),
+  slug: fake((f) => f.lorem.slug(3)),
+  taxCategory: null,
+  externalTaxRate: null,
+  shippingDetails: null,
+  priceMode: oneOf(...Object.values(customLineItemPriceMode)),
+  custom: null,
+};
+
+export const restFieldsConfig: TModelFieldsConfig<TCustomLineItemDraftRest> = {
+  fields: {
+    ...commonFieldsConfig,
+  },
+};
+
+export const graphqlFieldsConfig: TModelFieldsConfig<TCustomLineItemDraftGraphql> =
+  {
+    fields: {
+      ...commonFieldsConfig,
```

Comment:
for graphql, I think tax category needs to have taxCategoryRef as well and anything that could have reference expansion needs 2 fields for GraphQL one for reference expansion and one for reference, check out the Cart entity to see how the `postbuild` is done